src/App.jsx

```diff
@@ -238,6 +238,17 @@ const App = () => {
           newAuction['type'] = 'new'
         }
 
+        const currentProfileData = queryClient.getQueryData([
+          QUERY_KEYS.profile,
+          { userAddress: address },
+        ]);
+        if (newAuction.lister === address && currentProfileData) {
+          currentProfileData.listings = [data, ...currentProfileData.listings]
```

Comment:
you might be missing an id field in `data`. I got a console error that broke the page after an item was listed:

![image](https://user-images.githubusercontent.com/19379512/147789900-43111f4a-cd22-4a60-aea0-eefd1201c26f.png)

I think this happened because the itemListed event triggered an update for react-query cache

Reply:
Ok, I will remove that.